compiler/rustc_target/src/spec/mod.rs

```diff
@@ -2063,7 +2064,8 @@ pub struct TargetOptions {
     /// However, parts of the backend do check this field for specific values to enable special behavior.
     pub abi: StaticCow<str>,
     /// Vendor name to use for conditional compilation (`target_vendor`). Defaults to "unknown".
-    pub vendor: StaticCow<str>,
+    #[rustc_lint_opt_deny_field_access("use `Target::is_like_*` instead of this field")]
```

Comment:
Can you add a link to https://github.com/rust-lang/lang-team/issues/102 to explain why it's denied?

Reply:
Done (though I linked to the [tracking issue](https://github.com/rust-lang/rust/issues/100343) instead).